packages/tasks/src/local-apps.ts

```diff
@@ -117,10 +117,15 @@ const snippetLlamacpp = (model: ModelData, filepath?: string): LocalAppSnippet[]
 	};
 	return [
 		{
-			title: "Install from brew",
+			title: "Install from brew (MacOS)",
```

Comment:
```suggestion
			title: "Install from brew",
```

I have never tried it, but homebrew is also available on Linux (and Windows, I think)

Reply:
What about (MacOS, Linux) - the llama.cpp formulae is not available through brew on Windows - wdyt?

Reply:
i think most people know brew so no need to specify the platforms imo (only for winget)